openapi_test.go

```diff
@@ -201,6 +216,17 @@ func Test_tagFromType(t *testing.T) {
 			}
 		})
 	}
+
+	t.Run("struct with nested tags", func(t *testing.T) {
+		s := NewServer()
+		tag := SchemaTagFromType(s.OpenAPI, MyStructWithNested{})
+		nestedProperty := tag.Value.Properties["nested"]
+		require.Equal(t, "my struct", nestedProperty.Value.Description)
+		require.Equal(t, "my description", nestedProperty.Value.Properties["c"].Value.Description)
+		require.Equal(t, 8, nestedProperty.Value.Properties["c"].Value.Example)
+		require.Equal(t, float64(3), *nestedProperty.Value.Properties["c"].Value.Min)
+		require.Equal(t, float64(10), *nestedProperty.Value.Properties["c"].Value.Max)
```

Comment:
This is an abuse of using testify require package

You should use assert package here. require fails on first error, so it won't bring you good information about what is broken, you will have to fix them one by one

But I think the problem is more general

Reply:
updated for this case